lib/query/query_recurring_donations.rb

```diff
@@ -144,43 +158,50 @@ def self.for_export_enumerable(npo_id, query, chunk_limit=15000)
-    'MAX(campaigns.name) AS campaign_name',
-    'MAX(supporters.name) AS supporter_name',
-    'MAX(supporters.email) AS supporter_email',
-    'MAX(supporters.phone) AS phone',
-    'MAX(supporters.address) AS address',
-    'MAX(supporters.city) AS city',
-    'MAX(supporters.state_code) AS state',
-    'MAX(supporters.zip_code) AS zip_code',
-    'MAX(cards.name) AS card_name',
-    'recurring_donations.id AS "Recurring Donation ID"',
-    'MAX(donations.id) AS "Donation ID"',
-    'MAX(donations.designation) AS "Designation"',
-    'BOOL_OR(misc_recurring_donation_infos.fee_covered) AS "Fee Covered by Supporter"',
-    "CASE WHEN #{is_cancelled_clause('recurring_donations')} 
-      THEN 'cancelled' 
-    WHEN #{is_failed_clause('recurring_donations')} 
-      THEN 'failed' 
-    ELSE 'active' END AS status",
-    'recurring_donations.cancelled_at AS "Cancelled At"',
-    "CASE WHEN #{is_active_clause('recurring_donations')} OR #{is_failed_clause('recurring_donations')} THEN concat('#{root_url}recurring_donations/', recurring_donations.id, '/edit?t=', recurring_donations.edit_token) ELSE '' END AS \"Donation Management Url\"",
-    'MAX(recurring_donations.paydate) AS "Paydate"',
-    'MAX(paid_charges.created_at) AS "Last Charge Succeeded"'
-  ]
+    include_last_failed_charge = query[:include_last_failed_charge]
+    select_list = [
+      'recurring_donations.created_at',
+      '(recurring_donations.amount / 100.0)::money::text AS amount',
+      "concat('Every ', recurring_donations.interval, ' ', recurring_donations.time_unit, '(s)') AS interval",
+      '(SUM(paid_charges.amount) / 100.0)::money::text AS total_contributed',
+      'MAX(campaigns.name) AS campaign_name',
+      'MAX(supporters.name) AS supporter_name',
+      'MAX(supporters.email) AS supporter_email',
+      'MAX(supporters.phone) AS phone',
+      'MAX(supporters.address) AS address',
+      'MAX(supporters.city) AS city',
+      'MAX(supporters.state_code) AS state',
+      'MAX(supporters.zip_code) AS zip_code',
+      'MAX(cards.name) AS card_name',
+      'recurring_donations.id AS "Recurring Donation ID"',
+      'MAX(donations.id) AS "Donation ID"',
+      'MAX(donations.designation) AS "Designation"',
+      'BOOL_OR(misc_recurring_donation_infos.fee_covered) AS "Fee Covered by Supporter"',
+      "CASE WHEN #{is_cancelled_clause('recurring_donations')} 
+        THEN 'cancelled' 
+      WHEN #{is_failed_clause('recurring_donations')} 
+        THEN 'failed' 
+      ELSE 'active' END AS status",
+      'recurring_donations.cancelled_at AS "Cancelled At"',
+      "CASE WHEN #{is_active_clause('recurring_donations')} OR #{is_failed_clause('recurring_donations')} THEN concat('#{root_url}recurring_donations/', recurring_donations.id, '/edit?t=', recurring_donations.edit_token) ELSE '' END AS \"Donation Management Url\"",
+      'MAX(recurring_donations.paydate) AS "Paydate"',
+      'MAX(paid_charges.created_at) AS "Last Charge Succeeded"'
+    ]
```

Comment:
Please undo this reformatting. It's generally a good idea but it muddles the changes I think.